storage/engine.go

```diff
@@ -188,6 +192,14 @@ func (e *engine) Configure(config core.ServerConfig) error {
 			return fmt.Errorf("unable to configure redis client: %w", err)
 		}
 		e.sessionDatabase = NewRedisSessionDatabase(client, redisConfig.Database)
+		log.Logger().Info("Redis session storage support enabled.")
+	} else if memcachedConfig.isConfigured() {
+		memcachedClient, err := newMemcachedClient(memcachedConfig)
+		if err != nil {
+			return fmt.Errorf("unable to initialize memcached client: %w", err)
+		}
+		e.sessionDatabase = NewMemcachedSessionDatabase(memcachedClient)
+		log.Logger().Info("Memcached session storage support enabled.")
```

Comment:
add this line for Redis as well (or remove it here) (and consider it for in0-memory).

Reply:
already is